### ISSUE TITLE
[Enhancement] Sort Lists

### ISSUE THREAD

**Issue by rizreflects** [drinfernoo/plugin.program.autowidget#80]
When adding a trakt list as a widget through TMDB Helper, it gives you an option to order the list in anyway you want. So for example, I can set it to show movies from newest-to-oldest. This comes in handy for certain trakt lists that are ordered from A-Z or oldest-to-newest. However, I don't think this is currently doable using Autowidget. Would it be possible to add this feature? It would be nice if I could go into "my groups", click "edit" on the list I want and change the sort order.

**Comment by drinfernoo** [comment]
I have an issue open for implementing this functionality in AutoWidget, but it will work a bit differently from the built-in feature of TMDbHelper.

Essentially, you will be able to edit it via the editor dialog in AutoWidget, as you described, rather than when pointing the widget like TMDbHelper does.

**Comment by jmh2002** [comment]
That's awesome that you will be adding this, as I was just about to raise it again on the forum thread and ask if you could implement the TMDBH method too instead of rely on the skin for this.

Via the AW dialog should be fine too - in fact better, because I some AW's which I view directly as a list.

THANKS, I'm REALLY looking forward to this, since it's been limiting my use of merged sources which couldn't be sorted by date.

**Comment by drinfernoo** [comment]
Ah, I guess the issue I have open is regarding paged widgets. I do want to allow this to work, but it will take some work to pull it off, I think.

**Comment by jmh2002** [comment]
Well there are a few different 'sorting' related issues open, per path, paged, etc. Hope you are able to resolve them, but at a basic level sorting a list in general seems necessary when adding more than one source.

**Comment by drinfernoo** [comment]
@jmh2002 I agree. I intend to work on it at some point. Please hold for updates.

**Comment by jmh2002** [comment]
Thank you, most appreciated :)

**Comment by rizreflects** [comment]
@drinfernoo Great news. I'll be looking forward to the update. Thanks for all your work.

**Comment by Goldenfreddy0703** [comment]
Hey there, im also looking forward to this update too cause if i want to add my favorites, i got to make it easier to add instead of finding them in my my groups.

**Comment by Yuzuriha-San** [comment]
Nearly 2 years later, any updates?
